### PR TITLE
language alterations requested by marctjones

### PR DESCRIPTION
**Pull request by billwanjohi**
[//]: # (rtdbot-start)

URL of RTD document: https://civicactions-handbook.readthedocs.io/en/more-security-policy-edits/ Documentation Status

[//]: # (rtdbot-end)

### CODE REVIEW

File: docs/03-policies/security.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ CivicActions IT services provide a number of general user accounts. This include
 
 Usage of CivicActions user accounts should be as follows:
 
-- Usage must be directly related to your work with CivicActions - personal use (including personal projects) must be approved in advance by the CIO.
+- Usage must be directly related to your work with CivicActions - personal use (including personal projects) must be approved in advance by the CTO.
```
Comment (billwanjohi) [inline docs/03-policies/security.md:79]: https://github.com/CivicActions/handbook/pull/614#discussion_r474111503

File: docs/03-policies/security.md [hunk 2]
```diff
@@ -188,7 +188,7 @@ Developers and themers working on the site codebase (and committing code to Git)
   - Session hijacking, prevented by using SSL and correct site/session settings.
   - Data disclosure, prevented by carefully setting and testing access control, as well as using SSL as needed.
   - Password guessing attacks, mitigated by using strong passwords.
-- Avoid inclusion of any software not licensed under an approved CivicActions open source license on a project without prior approval from the legal team.
+- Software that is not licensed under an approved CivicActions open source license may not be used on a project without prior approval from the legal team.
```
Comment (billwanjohi) [inline docs/03-policies/security.md:191]: https://github.com/CivicActions/handbook/pull/614#discussion_r474109857

File: docs/03-policies/security.md [hunk 3]
```diff
@@ -198,15 +198,15 @@ Developers and themers maintaining local sandbox copies of client sites must als
 - Ensure that unsanitized mysql data (extracted via mysqldump or phpmyadmin) is not downloaded from the server to a local sandbox.
 - Ensure that all confidential data associated with a project (such as databases, database dumps and other files) are securely deleted from their system(s) when leaving or completing a project.
 
-Developers and themers working on the site vhost (SSH/shell, file system, database) must also:
+Developers and themers working on the site server instance (SSH/shell, file system, database) must also:
```
Comment (billwanjohi) [inline docs/03-policies/security.md:201]: https://github.com/CivicActions/handbook/pull/614#discussion_r474112516

**Review by openprivacy — APPROVED** [review]
Looks good to me - thanks!

**Pull request merged**